backend/docs/architecture.md

```
@@ -0,0 +1,372 @@
+- Releases funds when conditions are met
+- Can refund if deadline expires
+
+---
+
+# 3. Folder Structure
+
+## Frontend Structure
+
+apps/frontend/
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)